src/main/webapp/WEB-INF/web.xml

```diff
@@ -27,7 +27,7 @@
         <filter-class>co.refiere.resources.filter.AuthenticationFilter</filter-class>
         <init-param>
             <param-name>excludedPaths</param-name>
-            <param-value>/rest/v1/company/register,/rest/v1/plan/all,,/rest/v1/prize/all,/rest/database/2/import</param-value>
+            <param-value>/rest/v1/company/register,/rest/v1/plan/all,/rest/v1/prize/all,/rest/database/2/import</param-value>
```

Comment:
These 2 end-points should not be excluded: /rest/v1/prize/all and /rest/database/2/import